SmartTutor/SmartTutor.csproj

```diff
@@ -4,5 +4,17 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.9">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.9">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.4" />
```

Comment:
We might need the <PackageReference Include="Npgsql" Version="4.1.5" /> reference as well, but we can check when we get the DB working.

When you get pgAdmin up and running, see if the migrations created the suitable DB tables and add a few rows. Then make a dummy method in the Smart Tutor controller which accesses the appropriate DBSet and gathers the data. Start the app and access the controller through the browser. The link I sent you should get you up and running, but we can schedule a skype sink whenever you find the time (or talk on Thursday).